independent-publisher-connectors/Proposal - GIPHY (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,848 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "GIPHY",
+    "description": "This custom connector wraps around GIPHY API. It can be used to get GIFs related to a search string that is provided as input",

```

Comment:
@priyanshu870 Please remove the API word from the description and add a period at the end of it.